docs/python/testing.md

```diff
@@ -163,6 +163,25 @@ If discovery fails (for example, the test framework isn't installed or you have
 
 ![Discovery failure error messaged displayed in the Test Explorer](images/testing/test-discovery-error.png)
 
+### Django unittests
+
+The Python extension also offers support for running Django unittests! You get your Django tests discovered with only a few additional set up steps.
+
+First, ensure you have correctly set `"python.testing.unittestEnabled": true,` in your `settings.json` file, then follow the steps below to add `MANAGE_PY_PATH` as an environment variable:
+
+- Create a file named `.env` at the root of your project.
+- Add `MANAGE_PY_PATH='path-string-to-manange.py-path'` to the `.env` file, replacing `path-string-to-manange.py-path` with the path to your application's `manage.py` file (Note: you can copy the path by right clicking on the file in the File Explorer and selecting **Copy Path**).
+- Add  `"python.envFile": "${workspaceFolder}/.env"` to your `settings.json` file, so the Python extension can load the environment variables in this file when running and discovering tests (edit path as necessary if `.env` is not at root).
+- Add Django args to `"python.testing.unittestArgs": []` in `settings.json` as needed, and remove any arguments that are not compatible with Django.
+
+Navigate to the Testing Panel, and refresh the tree-view to have your Django tests displayed!
+
+In the case your tests are not discovered, try the following troubleshooting steps:
+
+- Search for error messages in the **Python** output panel.
+- Try running commands in the Command Line, then transfer the same command to how it is represented in VS Code. For example, you may run `python manage.py test --arg` in the Command Line which is represented by setting `MANAGE_PY_PATH='./manage.py'` and `"python.testing.unittestArgs": [--arg]` in VS Code. Check the output as it should print the commands that are actually ran.
```

Comment:
```suggestion
- Try running commands in the command line, then transfer the same command to how it is represented in VS Code. For example, you may run `python manage.py test --arg` in the command line, which is represented by setting `MANAGE_PY_PATH='./manage.py'` and `"python.testing.unittestArgs": [--arg]` in VS Code. Check the output as it should print the commands that are actually ran.
```

Reply:
That's a good point. The target users of this feature are more "power" users of Django, so they would likely know the commands to run in the terminal (which could be the one in VS Code or an external one, shouldn't matter). Translating it into VS Code would ideally be following the set up steps and leveraging the Python extension's testing args setting if needed. 

Based on this comment I made a couple of changes:
- Changed command line -> terminal
- Moved the whole Django subsection into its own section, similar to what we've done to highlight IntelliSense for pytest
- Added a troubleshooting subsection to the Django unittest section, adding a few more details and linking to the Django docs on how to run tests in the terminal